misk/src/main/kotlin/misk/digester/Windower.kt

```diff
@@ -0,0 +1,108 @@
+package misk.digester
+import java.util.Calendar
+/** Window represents a time range */
+class Window (private val start: Calendar, private val end: Calendar) {
```

Comment:
Use a data class instead, which gives you equals, toString, and hashCode for free.

```
data class Window (val start: Instant, val end: Instant) {
  fun contains(t: Instant): Boolean { ...  }
}
``` 

This also make `start` and `end` public.